conans/client/printer.py

```diff
@@ -197,7 +197,7 @@ def print_search_packages(self, packages_props, reference, recipe_hash, packages
         self._out.info("Existing packages for recipe %s:\n" % str(reference))
         # Each package
         for package_id, properties in sorted(packages_props.items()):
-            self._print_colored_line("Package_ID", package_id, 1)
+            self._print_colored_line("Package ID", package_id, 1)
```

Comment:
Though not pretty, some users might be relying on parsing the search output to get package IDs. I'd say, leave it as-is.

Reply:
Don't like "Package_ID" as it is not a parameter nor a variable, but I understand the reason